types/google-apps-script/google-apps-script.spreadsheet.d.ts

```diff
@@ -94,6 +94,32 @@ declare namespace GoogleAppsScript {
      * Styles that can be set on a range using Range.setBorder(top, left, bottom, right, vertical, horizontal, color, style).
      */
     enum BorderStyle { DOTTED, DASHED, SOLID, SOLID_MEDIUM, SOLID_THICK, DOUBLE }
+    /**
+     * Represents an image to add to a cell. To add an image to a cell, you must create a new image
+     * value for the image using SpreadsheetApp.newCellImage() and CellImageBuilder. Then you can
+     * use Range.setValue(value) or Range.setValues(values) to add the image value to the cell.
+     */
+    interface CellImage {
+      getAltTextDescription(): string;
+      getAltTextTitle(): string;
+      getContentUrl(): string;
+      getUrl(): string;
+      toBuilder(): CellImageBuilder;
+    }
+    /**
+     * Builder for CellImage. This builder creates the image value needed to add an image to a cell.
+     */
+    interface CellImageBuilder {
+      build(): CellImage;
+      getAltTextDescription(): string;
+      getAltTextTitle(): string;
+      getContentUrl(): string;
+      getUrl(): string;
+      setAltTextDescription(description: string): CellImage;
+      setAltTextTitle(title: string): CellImage;
```

Comment:
```suggestion
      setAltTextDescription(description: string): CellImageBuilder;
      setAltTextTitle(title: string): CellImageBuilder;
```

Google docs says these functions returns CellImage, but it returns CellImageBuilder in actual.
It might be erratta of google docs. Because other setter functions of XXX Builder returns XXX Builder (for method chain).

```js
// print CellImageBuilder
Logger.log(SpreadsheetApp.newCellImage().setAltTextDescription("foo"));

// print CellImage
Logger.log(SpreadsheetApp.newCellImage().setAltTextDescription("foo").build());
```

Reply:
I sent a feedback to Google about it.

Reply:
Oh, thank you. Also, another weird point in this [doc](https://developers.google.com/apps-script/reference/spreadsheet/cell-image-builder) is that it has a link `CellImage` in the top description that is currently point to `CellImageBuilder` (current page).